activejdbc-instrumentation/src/main/java/org/javalite/instrumentation/ModelInstrumentation.java

```diff
@@ -38,6 +38,7 @@ public ModelInstrumentation() throws NotFoundException {
 
         try {
             doInstrument(target);
+            target.detach();
```

Comment:
does not seem to be a problem. Since in most cases of using Maven this is irrelevant, and only affects Gradle, this seems as no risk. Will accept PL!